project/SparkRedshiftBuild.scala

```diff
@@ -57,6 +57,7 @@ object SparkRedshiftBuild extends Build {
       libraryDependencies ++= Seq(
         "com.amazonaws" % "aws-java-sdk-core" % "1.9.40",
         "com.amazonaws" % "aws-java-sdk-s3" % "1.9.40",
+        "com.amazonaws" % "aws-java-sdk-sts" % "1.9.40" % "test",
```

Comment:
FWIW, I just spun up an EC2 instance to try a couple of things out and it looks like it's pretty easy to obtain security tokens when the EC2 instance has an IAM profile:

```scala